cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
     }
 })
 
+// create login command
+y.scriptName("connectme")
+y.command({
+    command: 'login',
+    describe: 'login with an account',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        password: {
+            describe: 'Password of the account'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        login(argv)
+    }
+})
+
+// create guest account command
+y.scriptName("connectme")

```

Comment:
scriptName has already been used above, I have a feeling this will create a problem

Reply:
done